src/main/java/model/Car.java

```diff
@@ -0,0 +1,101 @@
+package model;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Objects;
+
+import static java.util.Arrays.*;
+import static java.util.stream.Collectors.toList;
+
+public class Car {
+    public static final int START_SCORE = 1;
+    private int position;
+    private final String name;
+
+    public Car(String name, int position) {
+        validate(name);
+        this.name = name;
+        this.position = position;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public int getPosition() {
+        return position;
+    }
+
+    public static List<Car> getCar(String[] carNameList) {
+        return stream(carNameList)
+                .map(carName -> new Car(carName, START_SCORE))
+                .collect(toList());
+    }
+
+
+    public static List<Car> createCar(String[] carNameList) {
+        return stream(carNameList)
+                .map(carName -> {
+                    return new Car(carName, START_SCORE);
+                })
+                .collect(toList());
+    }
+
+    public static void validate(String str) {
+        String[] carNameList = str.split(",");
+        HashSet<String> checker = new HashSet<>();
```

Comment:
가급적 하위 구현체보다는 아래와 같이 상위의 인터페이스 타입인 List로 타입을 선언하시는 것이 좋습니다. 

```suggestion
        Set<String> checker = new HashSet<>();
```

[[OOP] 우리는 왜 List list = new ArrayList(); 라고 쓰는가](https://www.itsmo.dev/interface-as-type/) 글을 참고해보시면 도움이될 것 같아요.